[Release] Hotfix 2.23.5 => 2.23.6 (patch)

guanbinrui opened the pull request:
MF-0000

guanbinrui committed:
chore: bump version to 2.23.6

UncleBill commented:
Please pick patch from https://github.com/DimensionDev/Maskbook/pull/11033 as well.

Commit:
fix: erc20 allowance comparison (#11033)


#Conflicts:
#	packages/shared/src/UI/components/EthereumERC20TokenApprovedBoundary/index.tsx

Commit:
fix: ci error
